library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -299,4 +299,17 @@ data class Conversations(
         }
         return hmacKeysResponse.build()
     }
+
+    fun allTopics(): List<String> {
```

Comment:
Could we call this something like `getPushTopics()`, to kill the possibility that integrators misuse this?